src/StatusQ/Popups/StatusModal.qml

```diff
@@ -43,7 +43,7 @@ QC.Popup {
     }
 
     contentItem: Column {
-        width: parent.width
+        width: parent ? parent.width : 0
```

Comment:
sometimes parent can be null?

Reply:
was when using the contentItem, switched back to use content now